.travis.yml

```diff
@@ -2,12 +2,19 @@ language: python
 sudo: false
 python:
   - "2.7"
+  - "3.4"
+  - "3.5"
 
 env:
   - DJANGO="Django==1.8.15"
   - DJANGO="Django==1.9.10"
   - DJANGO="Django==1.10.1"
 
+matrix:
+  include:
+    - python: "3.3"
+      env: DJANGO="Django==1.8.15"
```

Comment:
1.9 and 1.10 do not longer support Python 3.3, but 1.8 does. If you don't plan on supporting Python 3.3 at all, I can remove these lines.